roles/pool/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "pool_sv2"
-version = "0.1.3"
+version = "0.1.0"
```

why are we walking backwards in time?
```suggestion
version = "1.0.0"
```

---

https://crates.io/crates/pool_sv2/versions